homeassistant/components/ring/camera.py

```diff
@@ -145,8 +145,8 @@ async def async_update(self) -> None:
             self._attr_motion_detection_enabled = self._device.motion_detection
             self.async_write_ha_state()
 
-        if self._last_event is None:
-            return
+        if self._last_event is None:  # pragma: no cover
```

Comment:
We don't have to use pragma no cover.

Reply:
Those checks appear to be redundant as they can't be reached. I can delete them from the platform code, is that what you're suggesting?

Reply:
I'm suggesting that we shouldn't hide test coverage.

So if it is not reachable, we should remove it. If it is needed for typing, we can guard it as such by using TYPE_CHECKING. As a last resort, we can also just accept the fact these lines aren't covered.

../Frenck

Reply:
Now asserted behind TYPE_CHECKING